webapp/webpack.config.js

```diff
@@ -1,18 +1,23 @@
 const exec = require('child_process').exec;
 
 const path = require('path');
+const webpack = require('webpack');
 
 const PLUGIN_ID = require('../plugin.json').id;
 
 const NPM_TARGET = process.env.npm_lifecycle_event; //eslint-disable-line no-process-env
+const E2E_TESTING = process.env.E2E_TESTING; //eslint-disable-line no-process-env
 let mode = 'production';
 let devtool = '';
 if (NPM_TARGET === 'debug' || NPM_TARGET === 'debug:watch') {
     mode = 'development';
     devtool = 'source-map';
 }
 
-const plugins = [];
+const plugins = [
+    new webpack.DefinePlugin({__E2E_TESTING__: E2E_TESTING}),
```

Comment:
@mickmister trigger is done through a custom env var as we discussed.

any thoughts?

0/5 on the name